test/index.js

```diff
@@ -474,4 +473,107 @@ describe('HauteCouture', () => {
         });
     });
 
+    it('allows options to be used as a callback', (done) => {
+
+        const opCbSrv = new Hapi.Server();
+        opCbSrv.connection();
+
+        const opCbPlugin = (server, options, next) => {
+
+            HauteCouture()(server, (err) => {
+
+                if (err) {
+                    return next(err);
+                }
+                next();
+            });
+        };
+
+        opCbPlugin.attributes = { name: 'no-options-plugin' };
+
+        opCbSrv.register(opCbPlugin, (err) => {
+
+            if (err) {
+                return done(err);
+            }
+
+            expect(opCbSrv.registrations['no-options-plugin']).to.exist();
```

Comment:
This just tests that HC doesn't choke when options are omitted– not that it did anything else correctly.  Can this test some very basic functionality of HC as well, just to prove that it does something?